integration-test/src/it/setup/depMgmt5/Sample.groovy

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2012 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.maven.ext.depMgmt5
+
+import org.commonjava.maven.ext.core.groovy.BaseScript
+import org.commonjava.maven.ext.core.groovy.InvocationStage
+import org.commonjava.maven.ext.core.groovy.PMEBaseScript
+import org.commonjava.maven.ext.core.groovy.InvocationPoint
+
+@InvocationPoint(invocationPoint = InvocationStage.ALL)
```

Comment:
This whole directory is just a copy of  the `BOTH` test, so I am not sure it is necessary.

Reply:
I agree! I would really like to avoid adding more things to setup unless its really necessary. I would think you don't need much setup surely? Could all this be moved into the actual test?

Reply:
I will try. I think the reason was I was looking for an existing script with `BOTH` and the only one I found was in setup.

Reply:
That shouldn't preclude moving every to a test ; otherwise we are imposing the cost of a setup for no apparent reason.

Reply:
I moved the test out of setup.